### PR TITLE
docs: 🤔 button group order/stacking

### LINKED ISSUE #771: docs: 🤔 button group order/stacking
## Current behavior
[Link to Storybook](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/story/components-sd-dialog--extended-footer)

button group does not stack vertically on mobile viewport
Image

Also in button groups the primary button is shown on the left hand side.
Image

## Expected behavior

button group should per default stack vertically on mobile viewport and can be overriden if desired
https://www.figma.com/file/fPGhgNZv98U4H69Gu2tlWi/Button?type=design&node-id=971-4578&mode=design&t=jlonZFop4yXvVu9O-11

Also in button groups the primary button should always be shown on the ride hand side.
https://www.figma.com/file/ASD84ZPlImePMJ888oCyqK/Dialog?type=design&node-id=1635-1538&mode=design&t=i1BXWbflfibeWvYw-11

## Technical Information
This only pertains to the sample and not the component itself.

## DoR
- [x] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Closes  #771.

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

**Comment by coraliefeil** [comment]
There is no chromatic link I can follow to review changes.

**Review by saemik94 — APPROVED** [review]

**Comment by Vahid1919** [comment]
> There is no chromatic link I can follow to review changes.

@coraliefeil It should work now :)

File: packages/components/src/components/dialog/dialog.stories.ts [hunk 1]
```diff
@@ -154,16 +168,24 @@ export const ExtendedFooter = {
...
           ><p class="sd-paragraph">
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
             phasellus dui vel id. Velit in sed.
           </p>
           <h4 slot="headline" class="sd-headline sd-headline--size-3xl">Lorem Ipsum</h4>
+          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button>
           <sd-button slot="footer" class="w-full">Label</sd-button>
-          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button></sd-dialog
-        >
+        </sd-dialog>
```
Comment (karlbaumhauer) [inline packages/components/src/components/dialog/dialog.stories.ts:188]: @Vahid1919 I know that you did not change anything in the `PreventClosing` story but on this version published by chromatic, the buttons are not working anymore. can you pls have a look into it and either fix it or open up a bug ticket?!

**Review by karlbaumhauer — APPROVED** [review]
LGTM besides the comment on the `PreventClosing`story.

File: packages/components/src/components/dialog/dialog.stories.ts (hunk 1 above)
Comment (Vahid1919) [reply packages/components/src/components/dialog/dialog.stories.ts:188]: It seems like the buttons don't work when you are coming from the main story. Reloading it seems to make it work. I will write a ticket for this and see what we can do.

**Comment by coraliefeil** [comment]
Review done, pls reassign.

**Comment by coraliefeil** [comment]
Approved. Thanks a lot <3

**Pull request merged**